catalog/openshift-cert-manager-operator/bundle-v1.14.0.yaml

```diff
@@ -343,9 +343,9 @@ properties:
 relatedImages:
 - image: registry.redhat.io/cert-manager/cert-manager-operator-bundle@sha256:92339bb4cfeaa527bd5d68065f58001e05ff3c05af3416e8e1f9dc60734784d3
   name: ""
-- image: registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:2df1899f8c9f877cb9e1239b860f34c0e7a27d72d87b561fa79a5d7319699d59
+- image: registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:38a4fe3f2a096ccd3e42973517e94212d8ac39bf1c38992f7f90cb1c2c0fed8c
   name: cert-manager-operator-rhel9-2df1899f8c9f877cb9e1239b860f34c0e7a27d72d87b561fa79a5d7319699d59-annotation
-- image: registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:2df1899f8c9f877cb9e1239b860f34c0e7a27d72d87b561fa79a5d7319699d59
+- image: registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:38a4fe3f2a096ccd3e42973517e94212d8ac39bf1c38992f7f90cb1c2c0fed8c
```

Comment:
We haven't made a release yet from konflux for v1.14, and these are just a placeholders for now. Once we have a release created from konflux, the bot will update the sha digests of the latest images. Since we have turned off auto-release, these are useful to test the builds.